.travis.yml

```diff
@@ -3,10 +3,14 @@ python:
   - '2.7'
 services:
   - elasticsearch
+  - redis-server
 before_script:
   - sleep 10
 env:
-  - DATABASE_URL=postgres://postgres@localhost:5432/postgres
+  global:
```

Comment:
Why did you add global? We aren't in a build matrix, so shouldn't need to specify that.